doc/guides/commit-queue.md

```diff
@@ -0,0 +1,103 @@
+# Commit Queue
+
+> tl;dr: You can land Pull Requests by adding the `commit-queue` label to it.
+> This feature is still experimental.
+
```

Comment:
We should probably state that adding a label `implies` the `certify safe` by the collaborator check the same way it is explicitly done on CI.

Reply:
I think this makes sense for the other PR, but not for this one. We don't have a "certify safety" when landing PRs in our current process

Reply:
I guess it would be better to just make sure we check every requirement for PR landing then.

Reply:
We do the same checks collaborators already do today by calling `git node land`. Can you think of anything we might not be checking at the moment?